app/moves/index.js

```diff
@@ -20,43 +24,50 @@ const uuidRegex =
 // Define param middleware
 router.param('locationId', setFromLocation)
 router.param('date', setMoveDate)
+router.param('period', setPeriod)
+router.param('status', setStatus)
 
 // Define shared middleware
 router.use('^([^.]+)$', saveUrl)
 
 // Define routes
 router.get('/', redirectBaseUrl)
 router.get(
-  '/:date',
+  '/:period(week|day)/:date',
   protectRoute('moves:view:all'),
+  setDateRange,
   setMovesByDateAllLocations,
   setPagination,
   list
 )
 router.get(
-  `/:date/:locationId(${uuidRegex})`,
+  `/:period(week|day)/:date/:locationId(${uuidRegex})`,
   protectRoute('moves:view:by_location'),
+  setDateRange,
```

Comment:
I spotted below we can probably just call this on the `:date` param middleware seeing as we use it in all routes that contain `:date`. It doesn't look like we have much need for `moveDate` either now so it could be replaced completely with this. And it would reduce the need to duplicate this in all of these route definitions.